### PR TITLE
Readme modify

### PR DESCRIPTION
**Pull request by jmaicaaan**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,72 +1,67 @@
...
 
+This is inspired from the [Java - Struts Framework](https://javabeat.net/introduction-to-struts-actions/) which we customized to make an opinionated starter pack for the developer to kick off there development fast.
 
-# Overview
-  - This is inspired from the [Java - Struts Framework](https://javabeat.net/introduction-to-struts-actions/) which we customized to make an opinionated starter pack for the developer to kick off there development fast.
-  - We also promote the `Single Responsibility` principle as we believe that it will be easier to maintain and provide unit tests when it scales up.
+We also promote the `Single Responsibility` principle as we believe that it will be easier to maintain and provide unit tests when it scales up.
 
-    ![flow](https://jmaicaaan.github.io/express-starter-ts/images/flow.jpg)
+  ![flow](https://jmaicaaan.github.io/express-starter-ts/images/flow.jpg)
```
Comment (ciiidi) [inline README.md]: can we add this issue on next branch? then replace this the updated flow. But if we will put this on current master, this is ok now

File: README.md [hunk 2]
```diff
@@ -1,72 +1,67 @@
...
 
-# Repositories
-  - The repository is the one responsible for communicating with the database with specific validations and nothing else! A repository extends the Repository class provide by the typeorm. **The repository file can contain multiple methods but each method is doing only one thing.**
-  - The repository will be injected into a controller in order to communicate to the database. It is the responsibility of the repository to communicate to the database.
+To run the test suites, type in your terminal
 
-# Services
-  - A service is similar to repositories but instead of communicating to database it communicates to other 3rd party libraries (nodemailer, bcrypt, etc). It creates an abstraction to the 3rd party libraries to be injectable in the application. **The service file can contain multiple methods but each method is doing only one thing.**
-  - The service will be injected into a controller that communicate to the other libraries. Basically its a class that has methods.
+    npm run test
```
Comment (ciiidi) [inline README.md]: if we will put this on the next branch. this will change now.

**Comment by ciiidi** [comment]
This is ok for me now. Looks good. I've just been thinking that if we will put this on the next branch? or we will another issue for that? to map the new changes for next branch

File: README.md (hunk 1 above)
Comment (jmaicaaan) [reply README.md]: Nope, for now this is not included in the next branch. The next branch is only created this week and may cause some bugs still. We can alter this as soon we make the next branch official

**Pull request merged**